syntaxes/dune.json

```diff
@@ -1225,6 +1222,9 @@
             }
           },
           "patterns": [
+            { 
+              "include": "$self" 
+            },
             {
```

Comment:
I think the rest of this is unnecessary now since "$self" already recognizes a `rule`.

Reply:
Oh right, Thank you